<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenamerWidget</class>
 <widget class="QWidget" name="RenamerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>88</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="dirLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="selectBtn">
         <property name="text">
          <string>选择目录</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="textLineEdit"/>
       </item>
       <item>
        <widget class="QRadioButton" name="prependRadBtn">
         <property name="text">
          <string>追加</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="replaceRadBtn">
         <property name="text">
          <string>替换</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="replaceLineEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>重命名</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
